### PR TITLE
Bug fix 16201

### PR DESCRIPTION
**Pull request by TomSendrovich**
Thank you for your contribution.
Before submitting this PR, please make sure:
- [ ] PR description and commit message should describe the changes done in this PR
- [ ] Verify the PR is point to correct branch i.e. Release or Beta branch if the code fix is for specific release , else point it to master
- [ ] Latest Code from master or specific release branch is merged to your branch
- [ ] No unwanted\commented\junk code is included
- [ ] No new warning upon build solution
- [ ] Existing unit test cases are passed
- [ ] New Unit tests are added for your development
- [ ] Sanity Tests are successfully executed for Existing Functionality
- [ ] After creating pull request there should not be any conflicts
- [ ] Resolve codacy comments
- [ ] Builds and checks are passed before PR is sent for review
- [ ] Resolve code review comments

### CODE REVIEW

File: Ginger/GingerCoreNET/ActionsLib/General/ActSetVariableValue.cs [hunk 1]
```diff
@@ -121,7 +121,15 @@ public override void Execute()
             {
                 if (Var.GetType() == typeof(VariableString))
                 {
-                    ((VariableString)Var).Value = calculatedValue;
+                    if (calculatedValue.Contains("ERROR"))
+                    {
+                        Error = calculatedValue;
```
Comment (MeniKadosh1) [inline Ginger/GingerCoreNET/ActionsLib/General/ActSetVariableValue.cs:126]: this change will cause the Action to fail which is different behavior compared to before, action was failed due to wrong VE on 4.4?

Comment (TomSendrovich) [reply Ginger/GingerCoreNET/ActionsLib/General/ActSetVariableValue.cs:126]: no, it wasn't failing

Comment (MeniKadosh1) [reply Ginger/GingerCoreNET/ActionsLib/General/ActSetVariableValue.cs:126]: so we don't want to change it now for not breaching existing flows.
please cancel the PR.

**Comment by MeniKadosh1** [comment]
Not merging to keep current action behavior.

**Pull request closed without merging**